urllib3/connection.py

```diff
@@ -167,12 +163,12 @@ def __init__(self, sock, state_machine, method=None, url=None):
         self.headers = self.msg = None
 
         # from the Status-Line of the response
-        self.version = _UNKNOWN
-        self.status = _UNKNOWN
-        self.reason = _UNKNOWN
+        self.version = None
+        self.status = None
+        self.reason = None
 
-        self.length = _UNKNOWN          # number of bytes left in response
-        self.will_close = _UNKNOWN      # conn will close at end of response
+        self.length = None          # number of bytes left in response
```

Comment:
Yeah, this can almost certainly go away too. Another PR for that would be much appreciated.